При -rules.alert.resend-delay=29s, округляется до evaluationInterval (30s), при -rules.alert.resend-delay=30s внезапно
ставится равным стандартному значению - 1m, что триггерит все DMS проверки в madison.

prometheus issue: https://github.com/prometheus/prometheus/issues/5048
prometheus-operator issue: https://github.com/coreos/prometheus-operator/issues/2253

diff --git a/pkg/prometheus/statefulset.go b/pkg/prometheus/statefulset.go
index d36c3aca..a21ee02c 100644
--- a/pkg/prometheus/statefulset.go
+++ b/pkg/prometheus/statefulset.go
@@ -341,6 +341,7 @@ func makeStatefulSetSpec(p monitoringv1.Prometheus, c *Config, ruleConfigMapName
 			"-storage.tsdb.retention="+p.Spec.Retention,
 			"-web.enable-lifecycle",
 			"-storage.tsdb.no-lockfile",
+			"-rules.alert.resend-delay=29s",
 		)
 	default:
 		return nil, errors.Errorf("unsupported Prometheus major version %s", version)
